migrations/versions/c336664ee988_rename_catalogue_admin_column_to_.py

```diff
@@ -0,0 +1,29 @@
+"""Rename catalogue_admin column to cataloging_admin in permissions table.
+
+Revision ID: c336664ee988
+Revises: 564f2b578774
+Create Date: 2017-10-12 10:27:15.211065
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'c336664ee988'
+down_revision = '564f2b578774'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Rename 'catalogue_admin' column to 'cataloging_admin' in permissions table."""
+    with op.batch_alter_table('permissions', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('cataloging_admin', sa.Boolean(), nullable=False))
```

Comment:
Please make sure we're upgradeable with data in *permissions* table, non-destructively! :)

```python
batch_op.alter_column('catalogue_admin', new_column_name='cataloging_admin')
```

Same with cataloger/registrant.

I ran into trouble when attempting to upgrade from develop branch version,